Readme.md

```diff
@@ -1,32 +1,44 @@
+# trash
 
 [![Crates.io](https://img.shields.io/crates/v/trash.svg)](https://crates.io/crates/trash)
 [![Docs.rs](https://docs.rs/trash/badge.svg)](https://docs.rs/trash)
 
-## About
+Trash is a Rust library for moving files and folders to the OS's Trash or Recycle Bin.
 
-Trash is a Rust library that provides functionality to move files and folders to the operating system's Recycle Bin (or Trash or Rubbish Bin or what have you).
+Supported platforms:
 
-The library supports Windows, Mac and Linux.
+- Linux/*BSD: [The FreeDesktop.org Trash specification](https://specifications.freedesktop.org/trash-spec/trashspec-1.0.html)
+- macOS
+- Windows
 
 ## Usage
 
+Add the following to your Cargo.toml:
+
+```toml
+[dependencies]
+trash = "1.1.1"
+```
+
+## Example
+
 ```rust
-extern crate trash;
 use std::fs::File;
 
 fn main() {
     // Let's create and remove a single file
-    File::create("remove-me").unwrap();
-    trash::remove("remove-me").unwrap();
-    assert!(File::open("remove-me").is_err());
-
+    let file = "remove-me";
+    File::create(file).unwrap();
+    trash::remove(file).unwrap();
+    assert!(File::open(file).is_err());
+    
     // Now let's remove multiple files at once
-    let the_others = ["remove-me-too", "dont-forget-about-me-either"];
-    for name in the_others.iter() {
+    let multiple_files = ["remove-me-too", "dont-forget-about-me-either"];
+    for name in multiple_files.iter() {
         File::create(name).unwrap();
     }
-    trash::remove_all(&the_others).unwrap();
-    for name in the_others.iter() {
+    trash::remove_all(&multiple_files).unwrap();
+    for name in multiple_files.iter() {
```

Comment:
Wheter these are improvements is subjective and I don't think it's worth our time to discuss this.